### PR TITLE
[SWE-1514][SWE-1424][BRANDBOOK MODULE] Write Main colors screen

### PR DESCRIPTION
**Pull request by ArtemSuhoy**

### CODE REVIEW

File: src/app/domains/Color/components/ColorSimpleView/ColorSimpleView.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+ */
+const ColorSimpleView = (props) => {
+  const { color, name, hex, rgb, textColor } = props
+
+  // [COMPUTED_PROPERTIES]
+
+  return (
+    <ColorSimpleViewStyled color={color}>
+      <Box>
+        <Box pb={16}>
```
Comment (alexpassss) [inline src/app/domains/Color/components/ColorSimpleView/ColorSimpleView.js:21]: Margin bottom

File: src/app/domains/Color/__constants__/colorsViewConfig.js [hunk 2]
```diff
@@ -2,21 +2,24 @@ const COLORS_VIEW_CONFIG = {
...
     viewSize: 'lg'
   },
   DARK: {
     shade: 'var(--brandbook-module-brand-neutral)',
     name: 'Dark',
-    hex: '#253037' /** FIXME: Must be fixed in next PR */,
-    rgb: 'R37 G48 B55' /** FIXME: Must be fixed in next PR */,
+    color: 'dark',
+    hex: '#253037',
+    rgb: 'R37 G48 B55',
```
Comment (alexpassss) [inline src/app/domains/Color/__constants__/colorsViewConfig.js:22]: Object should be normalized. Same fields for each obj

File: src/app/domains/Color/components/ColorSimpleView/ColorSimpleView.styled.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
+import styled, { css } from 'styled-components'
+
+const COLOR_THEME = {
+  accent: css`
+    background-color: var(--brandbook-module-accent);
+  `,
+  white: css`
+    background-color: var(--brandbook-module-white);
+    border: 1px solid rgba(0, 0, 0, 0.16);
```
Comment (alexpassss) [inline src/app/domains/Color/components/ColorSimpleView/ColorSimpleView.styled.js:9]: Should be moved to the cars also

File: src/app/domains/Color/components/ColorSimpleView/ColorSimpleView.js [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+      </Box>
+    </ColorSimpleViewStyled>
+  )
+}
+
+ColorSimpleView.propTypes = {
+  color: PropTypes.string.isRequired,
+  name: PropTypes.string.isRequired,
+  hex: PropTypes.string.isRequired,
+  rgb: PropTypes.string.isRequired
```
Comment (AnastasiaShapowalova) [inline src/app/domains/Color/components/ColorSimpleView/ColorSimpleView.js:41]: not all propTypes, "textColor" missed

File: src/app/domains/Color/components/ColorSimpleView/ColorSimpleView.js [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+import ColorSimpleViewStyled from './ColorSimpleView.styled.js'
+import PropTypes from 'prop-types'
+
+/**
+ *
+ * @param color - color wrapper
+ * @param name - title in wrapper
+ * @param hex - description color in wrapper
+ * @param rgb - description color in wrapper
+ */
```
Comment (Olexandr25) [inline src/app/domains/Color/components/ColorSimpleView/ColorSimpleView.js:12]: You missed param textColor
Please write correct js doc
@param {string} textColor

**Review by alexpassss — APPROVED** [review]

File: src/app/domains/Color/__constants__/colorsViewConfig.js [hunk 6]
```diff
@@ -1,30 +1,31 @@
 import { getHexValue, getRgbValue } from '../helpers/getColorValue'
```
Comment (Olexandr25) [inline src/app/domains/Color/__constants__/colorsViewConfig.js:1]: write full path, to the file

File: src/app/screens/Color/ColorContent/MainColors/MainColorsScreen.js [hunk 7]
```diff
@@ -0,0 +1,20 @@
+import { ColorsList } from 'domains/Color/components'
+import { ContentScreen, PositionedContainer } from '~/app/components'
```
Comment (Olexandr25) [inline src/app/screens/Color/ColorContent/MainColors/MainColorsScreen.js:2]: rewrite all imports, to ~app/

File: src/app/domains/Color/components/ColorsList/ColorsList.js [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+import { ColorSimpleView } from '~/app/domains/Color/components'
+
+/**
+ * @returns color wrappers block (main colors screen - content).
+ */
+
+const ColorsList = () => {
+  return (
+    <Row noGutters>
+      <Col cw={12} h="right">
```
Comment (Anhelina4) [inline src/app/domains/Color/components/ColorsList/ColorsList.js:13]: Do you need cw={12} here?

**Review by whoiam460 — APPROVED** [review]

**Pull request merged**